core/Inst.hpp

```diff
@@ -32,6 +32,32 @@ namespace olympia
     class Inst {
     public:
 
+        class RenameData{
+            public:
+            void setDestination(unsigned int destination){
+                dest = destination;
+            }
+            void setOriginalDestination(unsigned int destination){
+                original_dest = destination;
+            }
+            void setSource(std::pair<unsigned int, bool> source){
```

Comment:
Consider making an alias for the `pair` type: 
```
    using SourceReg = std::pair<uint32_t, bool>;
    void setSource(const SourceReg & source) {
```
And replace all references to `pair` with this alias.  That way you can change the type in 1 place (if you decide to do so).

Reply:
Addressed in [163a70b](https://github.com/riscv-software-src/riscv-perf-model/pull/40/commits/163a70b5862658deca0df16b035bbe59dc841c92)